src/ProjectTemplates/test/SpaTemplateTest/AngularTemplateTest.cs

```diff
@@ -14,7 +15,7 @@ public AngularTemplateTest(ProjectFactoryFixture projectFactory, BrowserFixture
             : base(projectFactory, browserFixture, output) { }
 
         [Fact(Skip = "https://github.com/aspnet/AspNetCore-Internal/issues/1854")]
-        public void AngularTemplate_Works()
-            => SpaTemplateImpl("angular");
+        public async Task AngularTemplate_Works()
+            => await SpaTemplateImpl("angular");
```

Comment:
nit: I also made these async changes as part of my changes. You don't need to await here.

Reply:
What changes are these you're talking about?

Reply:
https://github.com/aspnet/AspNetCore/pull/8275/checks